pkg/cmd/root.go

```diff
@@ -83,3 +85,20 @@ func Root(p cli.Params) *cobra.Command {
 
 	return cmd
 }
+
+func hasMainSubCommands(cmd *cobra.Command) bool {
+	return len(subCommands(cmd, "main")) > 0
+}
+func hasUtilitySubCommands(cmd *cobra.Command) bool {
+	return len(subCommands(cmd, "utility")) > 0
+}
+
+func subCommands(cmd *cobra.Command, annotation string) []*cobra.Command {
+	cmds := []*cobra.Command{}
+	for _, sub := range cmd.Commands() {
+		if sub.IsAvailableCommand() && sub.Annotations["commandType"] == annotation {
+			cmds = append(cmds, sub)
```

Comment:
can return `true` instead of populating any array, given the fact `cmds` are not used any where?

Reply:
@hrishin maybe :angel: I was thinking we may want to use this later on and need to array but…

Reply:
Ahh was also wondering on the same if we can use it later.

Cool, nothing much to worry about it :smile: